Astro 3/26 release notes (#3559)
astronomer/docs: astro/release-notes.md

<!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->

## March 26, 2024

### Refactored Astro API documentation

Astro API documentation is now hosted at [https://docs.astronomer.io/api](https://docs.astronomer.io/api). In the new API documentation center, you can:

- Format and test API requests directly in your browser.
- Export requests to Python, Javascript, and curl.
- View weekly changelogs for the API.

### Create Deployments with deprecated versions of Astro Runtime

> You can now use the Astro API to create Deployments with deprecated versions of Astro Runtime. This option is necessary if you're migrating existing Airflow environments to Astro or if you need to maintain deprecated environments for testing purposes. 

Author: ```suggestion
You can now use the Astro API to create Deployments with deprecated versions of Astro Runtime. Using deprecated Astro Runtime versions is sometimes necessary if you're migrating existing Airflow environments to Astro, or if you need to maintain deprecated environments for testing purposes. 
```

> Note that this feature is disabled by default. To use this feature, reach out to your account team and request for the feature to be enabled.

Author: ```suggestion
Note that this feature is disabled by default. To use this feature, reach out to your account team and request for the feature to be enabled. See [Run a deprecated Astro Runtime version](https://docs.astronomer.io/astro/upgrade-runtime#run-a-deprecated-astro-runtime-version) for more information.
```

> Note that this feature is disabled by default. To use this feature, reach out to [Astronomer support](https://cloud.astronomer.io/open-support-request) and request for them to enable the feature.

Author: ```suggestion
Note that this feature is disabled by default. To use this feature, reach out to your account team and request for the feature to be enabled.
```

### New GCP database instance types available

<HybridBadge/>

You can now use the following node instance types for database instances in GCP clusters:

- XLarge Compute Optimized (24 CPU, 48 GiB MEM)
- XXLarge Compute Optimized (32 CPU, 64 GiB MEM)
  

> See [GCP Hybrid cluster settings](resource-reference-gcp-hybrid.md#supported-cloud-sql-instance-types) for a list of all available database instance types.

Author: ```suggestion
See [GCP Hybrid cluster settings](https://docs.astronomer.io/astro/resource-reference-gcp-hybrid#supported-cloud-sql-instance-types) for a list of all available database instance types.
```

### Additional improvements

> - The Astro UI now includes tutorials for features that are not yet configured within your Organization.

Author: ```suggestion
- The Astro UI now includes [Learning Bytes](https://academy.astronomer.io/learning-bytes-reporting) for features that are not yet configured within your Organization.
```
